### PR TITLE
Update build.ps1 to support version 1.0.0

### PR DESCRIPTION
**Pull request by EdgardosReis**
based on documentation here https://cakebuild.net/docs/running-builds/runners/cake-runner-for-dotnet-framework#switches
changed also the other switches to "--". Not sure what to do with the -configuration, -experimental and -mono. Didn't change for now. tell me if this should be changed too.

### CODE REVIEW

**Comment by devlead** [comment]
You could probably take the bootstrappers from
https://www.github.com/cake-build/resources/tree/master/dotnet-framework

Mono and experimental no longer a thing, so those aren't needed in build.ps1/sh.

File: build.ps1 [hunk 1]
```diff
@@ -219,11 +219,11 @@ if (!(Test-Path $CAKE_EXE)) {
...
-if ($Target) { $cakeArguments += "-target=$Target" }
+if ($Target) { $cakeArguments += "--target=$Target" }
 if ($Configuration) { $cakeArguments += "-configuration=$Configuration" }
-if ($Verbosity) { $cakeArguments += "-verbosity=$Verbosity" }
-if ($ShowDescription) { $cakeArguments += "-showdescription" }
-if ($DryRun) { $cakeArguments += "-dryrun" }
+if ($Verbosity) { $cakeArguments += "--verbosity=$Verbosity" }
+if ($ShowDescription) { $cakeArguments += "--showdescription" }
+if ($DryRun) { $cakeArguments += "--dryrun" }
 if ($Experimental) { $cakeArguments += "-experimental" }
```
Comment (augustoproiete) [inline build.ps1]:
```suggestion
if ($Experimental) { $cakeArguments += "--experimental" }
```

File: build.ps1 [hunk 2]
```diff
@@ -219,11 +219,11 @@ if (!(Test-Path $CAKE_EXE)) {
 
 # Build Cake arguments
 $cakeArguments = @("$Script");
-if ($Target) { $cakeArguments += "-target=$Target" }
+if ($Target) { $cakeArguments += "--target=$Target" }
 if ($Configuration) { $cakeArguments += "-configuration=$Configuration" }
```
Comment (augustoproiete) [inline build.ps1]:
```suggestion
if ($Configuration) { $cakeArguments += "--configuration=$Configuration" }
```

**Review by augustoproiete — CHANGES_REQUESTED** [review]
Thanks for the PR @EdgardosReis!

With Cake v1.0 all input arguments must now use one `-` when passing a single character argument, and two `--` when using an argument that has multiple characters ~, so both `-configuration` and `-experimental` need to be updated as well~

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/cake-build/example?pullRequest=57) <br/>All CLA requirements met.

**Review by devlead — APPROVED** [review]
LGTM 👍

**Comment by devlead** [comment]
@EdgardosReis your changes have been merged, thanks for your contribution 👍

**Pull request merged**